scripts/install/usr/libexec/cortx-motr/motr-service.functions

```diff
@@ -565,6 +566,30 @@ m0_get_core_params()
         optional_params+=" -J $MOTR_M0D_SNS_BUFFER_POOL_SIZE"
     fi
 
+    if [[ -n "$MOTR_MOD_BTREE_TRICKLE_RELEASE" ]]; then
+        optional_params+=" -t $MOTR_MOD_BTREE_TRICKLE_RELEASE"
+    else
+        optional_params+=" -t $inv_val"
```

Comment:
No need to add invalid (or default) value if parameter is not set in the config file.
Same comment for other WM variables in this function.

Reply:
Done.